G2P.pm

```diff
@@ -1073,7 +1089,7 @@ sub _vcf_frequency_filtering {
   my $self = shift;
   my $tva = shift;
   my $allele = $tva->base_variation_feature->alt_alleles;  
-  my $vf = $tva->base_variation_feature;
+  my $vf = $tva->variation_feature;
```

Comment:
Why was this changed to `$tva->variation_feature`?

Reply:
Because `get_all_Alleles_by_VariationFeature ` in L1098 requires a `variation_feature object `

Reply:
Because `get_all_Alleles_by_VariationFeature ` in L1098 requires a `variation_feature `object

Reply:
This can interfere with alleles match.
`$tva->base_variation_feature` has the alleles before they are shifted. Some time ago we updated all plugins to fetch the alleles from base_variation_feature to make sure we were not using shifted alleles in plugins.
I'm not completely sure how much is affected if we call variation_feature here.

Reply:
Yeah I remember, not sure it affects here but I have switched it back to `base_variation_feature`
Thanks Diana